.github/workflows/ci.yml

```diff
@@ -12,11 +12,11 @@ jobs:
   test:
     name: Lint & Test Build
     if: github.event_name == 'pull_request'
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     container: node:16.17-alpine
```

Comment:
@Fallenbagel does this need to be changed to a newer version or not I was not sure?

Reply:
node:17-alpine maybe this one ?